addons/hw_drivers/websocket_client.py

```diff
@@ -13,19 +13,19 @@
 _logger = logging.getLogger(__name__)
 websocket.enableTrace(True, level=logging.getLevelName(_logger.getEffectiveLevel()))
 
-def send_to_controller(print_id, device_identifier):
+def send_to_controller(device_type, params):
+    routes = {
+        "printer" : "/iot/printer/status",
+        }
     server = helpers.get_odoo_server_url()
     try:
         urllib3.disable_warnings()
         http = urllib3.PoolManager(cert_reqs='CERT_NONE')
         http.request(
             'POST',
-            server + "/iot/printer/status",
+            server + routes.get(device_type, ''),
```

Comment:
does it make sense to have '' if the value is not provided ? Maybe it would be better to use `routes[device_type]` so that if it is is not provided we have a keyerror rather than a call to the server on a probably non-existing route (or even worst, a route that exists and will confuse the developer regarding the response)

Reply:
Yeah, good idea !